seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1741,7 +1741,7 @@
       "description": [
         "Sometimes you want to add <code>a</code> elements to your website before you know where they will link.",
         "This is also handy when you're changing the behavior of a link using <code>jQuery</code>, which we'll learn about later.",
-        "The <code>href</code> attribute is the quoted URL link within the <code>a</code> element. Replace the link value of your <code>a</code> element's <code>href</code> attribute with a <code>#</code>, also known as a hash symbol, to turn it into a dead link."
+        "The current value of the <code>href</code> attribute is a link that points to \"http://www.freecatphotoapp.com\". Replace this link with a <code>#</code>, also known as a hash symbol, to create a dead link."
```

Comment:
~~Replace this link with a <code>#</code>~~

Replace the <code>href</code> attribute value with a <code>#</code>